### PR TITLE
removed boolean check for running dev server in https

### PR DESCRIPTION
**Pull request by reznord**
Removed boolean check for running dev server in https from [here](https://github.com/developit/preact-cli/blob/2bd02cb8e5c442c0987d90d6cada3a91516fa790/src/lib/webpack/webpack-client-config.js#L126)

Now you can run dev server in `HTTPS`

Fix for bug #175

### CODE REVIEW

File: src/commands/watch.js [hunk 1]
```diff
@@ -19,8 +19,7 @@ export default asyncCommand({
 		},
 		host: {
 			description: 'Hostname to start a server on',
-			default: '0.0.0.0',
-			alias: 'h'
```
Comment (reznord) [inline src/commands/watch.js]: removing the alias for the `host` command since this is colliding with `yargs` package's `--help, -h`.

Comment (reznord) [inline src/commands/watch.js]: and this is supposed to be a separate PR, but since already a PR is in queue, it just appended !! 

Sorry for the mess 😅

Comment (thangngoc89) [inline src/commands/watch.js]: Please move this to a seperate PR as your PR could be squashed to create a clean git history.
